### ISSUE TITLE
Customisation of 'Action Trouble Text'

### ISSUE THREAD

**Issue by fridolin-koch** [matcornic/hermes#7]
Currently I can't find any possibility to change the 'action trouble text' inside the footer:

```
If you’re having trouble with the button 'Hier Anmelden', copy and paste the URL below into your web browser.
```

This makes it impossible to fully customise the email template (e.g. i18n).

I would propose the following changes: https://github.com/fridolin-koch/hermes/commit/44be3778410a55d7aa5f6951118d4c70de9cc0ad

I've you are happy with the proposed solution, I can sent a PR.

Best Regards,
Frido

**Comment by matcornic** [comment]
Hey @fridolin-koch ,

Thanks for your message, it's a nice idea ! Implementation looks great to me, I see a little edge case though.
Code does not check that user set a single `%s` when he overrides his *troubletext* string. The user may get weird behaviour without knowing it, because the email template expects a parameter to the `printf` function.

On top of that, we would just need more additions for the PR, could you:

- Update `README.md` with the new feature
- Update `CONTRIBUTING.md` with the new feature

Thank you :)

**Comment by fridolin-koch** [comment]
I agree, I can think of two possible solutions.

1. Check if the string contains `%s` exactly once, but this may also lead to unexpected behaviour in case the string contains `%d` `%b` etc.

2. Define a custom placeholder like `{ACTION}` and [replace](http://masterminds.github.io/sprig/strings.html#replace) with the action button text.

I think solution 2. would be less error prone. What do you think?

**Comment by matcornic** [comment]
I agree, let's go for the second solution ;)

**Comment by fridolin-koch** [comment]
I've created a PR (#8), let me know if anything is missing :)

**Comment by matcornic** [comment]
closed with #8
